### PR TITLE
526: Move wizard to /start

### PR DESCRIPTION
**Pull request by Mottie**
## Description

Per the new IA recommendations, the form wizards shouldn't share the `/introduction` path with the introduction page content. They are to be moved to a new `/start` path ([ref](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/information-architecture/ia-reviews/wizard-disability.md)).

This PR also needed to update the Cypress form tester to ignore the `/start` path.

Related:
- https://github.com/department-of-veterans-affairs/va.gov-team/issues/26264
- https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/information-architecture/ia-reviews/wizard-disability.md
- https://dsva.slack.com/archives/CBU0KDSB1/p1624386916190700?thread_ts=1624305807.150600&cid=CBU0KDSB1

## Testing done

- Updated unit tests
- Updated Cypress tests for wizard & introduction page

## Screenshots

N/A - The visual appearance hasn't changed, only the path

## Acceptance criteria
- [x] Wizard flow matches IA recommendations
- [x] All tests passing

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by ashagross — APPROVED** [review]

File: src/applications/disability-benefits/all-claims/Form526EZApp.jsx [hunk 1]
```diff
@@ -1,16 +1,14 @@
-import React, { useState, useEffect } from 'react';
+import React, { useEffect } from 'react';
 import { connect } from 'react-redux';
-import Breadcrumbs from '@department-of-veterans-affairs/component-library/Breadcrumbs';
+// import moment from 'moment';
```
Comment (k80bowman) [inline src/applications/disability-benefits/all-claims/Form526EZApp.jsx:3]: did you mean to leave this commented out?

File: src/applications/disability-benefits/all-claims/tests/containers/WizardContainer.unit.spec.jsx [hunk 2]
```diff
@@ -5,29 +5,32 @@ import { shallow } from 'enzyme';
...
-    setWizardStatus: value => {
-      global.status = value;
-    },
-  };
+  beforeEach(() => {
+    sessionStorage.removeItem(WIZARD_STATUS);
+  });
+  after(() => {
+    sessionStorage.removeItem(WIZARD_STATUS);
+  });
```
Comment (k80bowman) [inline src/applications/disability-benefits/all-claims/tests/containers/WizardContainer.unit.spec.jsx:16]: would it make more sense to put this all in an afterEach? Is there concern that sessionStorage will contain the Wizard Status from a test outside this file?

File: src/applications/disability-benefits/all-claims/Form526EZApp.jsx (hunk 1 above)
Comment (Mottie) [reply src/applications/disability-benefits/all-claims/Form526EZApp.jsx:3]: oops, yeah I should remove that

File: src/applications/disability-benefits/all-claims/tests/containers/WizardContainer.unit.spec.jsx (hunk 2 above)
Comment (Mottie) [reply src/applications/disability-benefits/all-claims/tests/containers/WizardContainer.unit.spec.jsx:16]: I didn't have a concern about sessionStorage outside of this test, but I figured it'd be safer to cleanup. Would you recommend using `afterEach` then?

Comment (k80bowman) [reply src/applications/disability-benefits/all-claims/tests/containers/WizardContainer.unit.spec.jsx:16]: Typically it's best to establish the pattern of cleaning up in an `afterEach`. However, I understand wanting to be sure that your environment is clean before your tests are run, so I'll leave it up to you. If you want to keep it as is, that's fine.

**Review by k80bowman — APPROVED** [review]

**Pull request merged**